webapp/client/src/pages/CareCostsInfoPage.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+import InfoBox from "../components/InfoBox";
+import StepHeaderButtons from "../components/StepHeaderButtons";
+// eslint-disable-next-line import/named
+import { anchorBack, anchorList } from "../lib/contentPagesAnchors";
+import {
+  ContentWrapper,
+  List,
+  AnchorListItem,
+  ListItem,
+  Headline1,
+  Headline2,
+  Paragraph,
+  ParagraphLarge,
+} from "../components/ContentPagesGeneralStyling";
+
+export default function CareCostsInfoPage() {
+  const { t } = useTranslation();
+
+  const anchorListItemsMap = anchorList
+    .filter((item) => item.text !== "Pflegekosten")
+    .map((anchor) => (
+      <AnchorListItem key={anchor.text}>
+        <a href={anchor.url}>{anchor.text}</a>
+      </AnchorListItem>
+    ));
+
+  const listCareCostExamples = [
+    {
+      text: t("CareCostsInfo.Paragraph2.ListItem1"),
+    },
+    {
+      text: t("CareCostsInfo.Paragraph2.ListItem2"),
+    },
+  ];
+
+  const listCareCostItemsMap = listCareCostExamples.map((item) => (
+    <ListItem key={item.text}>{item.text}</ListItem>
+  ));
+
+  return (
+    <>
+      <ContentWrapper>
+        <StepHeaderButtons text={anchorBack.text} url={anchorBack.url} />
+        <Headline1>{t("CareCostsInfo.Paragraph1.Heading")}</Headline1>
+        <ParagraphLarge>{t("CareCostsInfo.Paragraph1.Text")}</ParagraphLarge>
```

Comment:
Because of the style adjustments for content pages I have changed this to ParagraphLarge in your PR.